consensus/obcpbft/viewchange.go

```diff
@@ -413,6 +413,8 @@ func (instance *pbftCore) processNewView2(nv *NewView) error {
 		instance.persistQSet()
 	}
 
+	instance.updateViewChangeSeqNo()
```

Comment:
The new-view assigns all sequence numbers that were mentioned in view-change messages.  The primary could have sent a new-view without having received the view-change from the previous primary.  I think we introduced this issue by deleting "trailing" null requests from the new-view message.  We can either re-introduce those null requests (meaning lots of messages on view change), or we could reset `seqNo` to `max(xSet)`.  What do you prefer?